test/integ/pandas/test_arrow_pandas.py

```diff
@@ -65,9 +65,7 @@ def test_scaled_tinyint(conn_cnx):
     cases = ["NULL", 0.11, -0.11, "NULL", 1.27, -1.28, "NULL"]
     table = "test_arrow_tiny_int"
     column = "(a number(5,2))"
-    values = (
-        "(" + "),(".join(["{}, {}".format(i, c) for i, c in enumerate(cases)]) + ")"
-    )
+    values = ",".join([f"({i}, {c})" for i, c in enumerate(cases)])
```

Comment:
This is just a small thing that caught my eye and noticed it could be a little simpler